plugins/by-name/lsp-format/default.nix

```diff
@@ -28,6 +28,10 @@ lib.nixvim.plugins.mkNeovimPlugin {
   ];
 
   description = ''
+    A wrapper around Neovims native LSP formatting.
+
+    ---
```

Comment:
Again, is this a common pattern in nixvim? I see you've added 22 of these :grin: